src/portfolio/PortfolioTable.js

```diff
@@ -12,9 +13,27 @@ const PortfolioTable = ({compact, contracts, proposals, onViewDetails}) => {
 		<table>
 			<thead>
 				<tr>
-					<th>Ref.</th>
-					<th>Purchase</th>
-                    <th>Indicative</th>
+					<th>
+						<FormattedMessage
+							id="ref"
+							description=""
```

Comment:
i think id is needed, description is probably not, plan to clear all of them after I confirm weblate does not use it